src/WalkerNavMenuEditStatus.php

```diff
@@ -17,13 +17,22 @@ public function start_el(&$output, $item, $depth = 0, $args = [], $id = 0) {
 
     $item_id = esc_attr($item->ID);
     $status = get_post_meta($item_id, '_menu_item_status', TRUE);
-    $status = $status !== '' ? (int) $status : 1;
+    $link_status = get_post_meta($item_id, '_menu_item_link_status', TRUE);
     $form_item = '
           <p class="field-checkbox description description-thin">
             <label for="edit-menu-item-status-' . $item_id . '">
               <input type="hidden" name="menu-item-status[' . $item_id . ']" value="1" />
               <input type="checkbox" id="edit-menu-item-status-' . $item_id . '" class="edit-menu-item-status" name="menu-item-status[' . $item_id . ']" value="0"' . checked($status, 0, FALSE) . ' />
-                ' . __('Hidden') . '
+                ' . __('Hidden', Plugin::L10N) . '
+            </label>
+          </p>
+';
+    $form_item .= '
+          <p class="field-checkbox description description-thin">
+            <label for="edit-menu-item-link-status' . $item_id . '">
+              <input type="hidden" name="menu-item-link-status[' . $item_id . ']" value="1" />
+              <input type="checkbox" id="edit-menu-item-link-status-' . $item_id . '" class="edit-menu-item-link-status" name="menu-item-link-status[' . $item_id . ']" value="0"' . checked($link_status, 0, FALSE) . ' />
+                ' . __('Remove anchor tag', Plugin::L10N) . '
```

Comment:
_"Remove anchor tag"_ is too technical (who knows what an "anchor tag" is?); it's clearly visible in the German translation that literally expresses the anchor as a (boat's) anchor instead of a link.  Also, you can only remove something that is there already; but this feature toggles the output in the first place.

If the first checkbox is "Hidden" (an adjective), then the second should also be an adjective; e.g., "Unlinked"

"Unlinked" is not really a word though.  Perhaps:

- [ ] Hidden
- [ ] No link

Or perhaps even more precise _"Only text, omit link"_

Or that, but shorter:

- [ ] Hidden
- [ ] Omit link

German: "Link weglassen"

Hm. Technically, "omit" sounds like the whole item would be skipped, not clear enough.

- [ ] Hidden
- [ ] Without link

German: "Ohne Link"

I'd go with that.